cup_of_join.py

```diff
@@ -0,0 +1,15 @@
+def cup_of_join(*args, sep='-'):
+    res = []
+
+    if args is None:
+        print(res)
+
+    for value in args:
+        res.extend(value)
+        res.append(sep)
+
+    res.pop()
+    print(res)
```

Comment:
Use returns, dont coupe input output operations (like print and input) with code logic. Print it outside